web/pages/program/[slug].tsx

```diff
@@ -148,11 +148,32 @@ const SessionRoute = ({
     currentSessionInProgram?._id,
     sessions
   );
+  const hasAssociatedProgram = startDateTime && currentSessionInProgram;
   const hasHighlightedSpeakers =
     speakers?.length === 1 || speakers?.length === 2;
+  const formattedStartDate = formatDateWithDay(start, mainVenueTimezone, ', ');
+  const formattedTimezone = getNonLocationTimezone(
+    start,
```

Comment:
As far as I can tell, because these don't check for `hasAssociatedProgram`, factoring them out here as introduced an error in the hypothetical case where `startDateTime` is falsy (in which case `sessionStart` returns `null`).

> RangeError: Invalid time value

Not saying that I've checked that the code handles nulls/missing data in all other cases, but it seems like a step backwards.

Reply:
cfc3720.